RULES.md

```diff
@@ -1,536 +1,1037 @@
+| [`AttributionWithoutRoleNotice`](#AttributionWithoutRoleNotice)                   	| Attribution with no role.                                                                                                                                   	| attribution_without_role                              |
+| [`DuplicateRouteNameNotice`](#DuplicateRouteNameNotice)                           	| Two distinct routes have either the same `route_short_name`, the same `route_long_name`, or the same combination of `route_short_name` and `route_long_name`. | duplicate_route_name                                  |
+| [`EmptyRowNotice`](#EmptyRowNotice)                                               	| A row in the input file has only spaces.                                                                                                                      | empty_row                                             |
+| [`EqualShapeDistanceSameCoordinatesNotice`](#EqualShapeDistanceSameCoordinatesNotice) | Two consecutive points have equal `shape_dist_traveled` and the same lat/lon coordinates in `shapes.txt`.                                                     | equal_shape_distance_same_coordinates                 |
+| [`FastTravelBetweenConsecutiveStopsNotice`](#FastTravelBetweenConsecutiveStopsNotice) | A transit vehicle moves too fast between two consecutive stops.                                                                            	                | fast_travel_between_consecutive_stops                 |
+| [`FastTravelBetweenFarStopsNotice`](#FastTravelBetweenFarStopsNotice)                 | A transit vehicle moves too fast between two far stops.                                                                            	                        | fast_travel_between_far_stops                         |
+| [`FeedExpirationDateNotice`](#FeedExpirationDateNotice)                           	| Dataset should be valid for at least the next 7 days. Dataset should cover at least the next 30 days of service.                                            	| feed_expiration_date                                  |
+| [`FeedInfoLangAndAgencyMismatchNotice`](#FeedInfoLangAndAgencyLangMismatchNotice) 	| Mismatching feed and agency language fields.                                                                                                                	| feed_info_lang_and_agency_mismatch                    |
+| [`InconsistentAgencyLangNotice`](#InconsistentAgencyLangNotice)                   	| Inconsistent language among agencies.                                                                                                                       	| inconsistent_agency_lang                              |
```

Comment:
Some of these hyperlinks seem to be broken when I click on them - for example:
* [InconsistentAgencyLangNotice](https://github.com/MobilityData/gtfs-validator/blob/6c5a72c58a0cbb8cce4df8f2a96900013400e312/RULES.md#InconsistentAgencyLangNotice)
* [LeadingOrTrailingWhitespacesNotice](https://github.com/MobilityData/gtfs-validator/blob/6c5a72c58a0cbb8cce4df8f2a96900013400e312/RULES.md#LeadingOrTrailingWhitespacesNotice)
* [MissingFeedInfoDateNotice](https://github.com/MobilityData/gtfs-validator/blob/6c5a72c58a0cbb8cce4df8f2a96900013400e312/RULES.md#MissingFeedInfoDateNotice)
* [MissingFeedInfoDateNotice](https://github.com/MobilityData/gtfs-validator/blob/6c5a72c58a0cbb8cce4df8f2a96900013400e312/RULES.md#MissingFeedInfoDateNotice)

...and more below that.